src/main/java/org/openhab/tools/analysis/checkstyle/BundleVendorCheck.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
I think that the license headers in openHab are now updated to 2017: https://github.com/openhab/openhab2-addons/pull/1865. Can you please change it, so that we follow the same pattern?